deploy/crds/metal3.io_baremetalhosts_crd.yaml

```diff
@@ -172,6 +172,61 @@ spec:
             online:
               description: Should the server be online?
               type: boolean
+            raid:
+              description: RAID configuration for bare metal server
+              properties:
+                cleanFlag:
+                  description: if cleanFlag is true will remove old configuration
+                    and set new configuration
```

Comment:
Let's mention the default

Reply:
I think, we need keep the old configuration, if user not set this flag. Avoid deleting old configurations without the user's knowledge.